cypress/support/testData.js

```diff
@@ -0,0 +1,5 @@
+export const user = {
+    userName: 'B@ry!',
+    password: 'B@ry!123'

```

Comment:
Consider using a more descriptive password for testing purposes, such as 'TestPassword123' or 'CypressPassword123', to improve code readability 📖